WordPress/Classes/ViewRelated/Media/ImageCropViewController.swift

```diff
@@ -10,7 +10,9 @@ class ImageCropViewController: UIViewController, UIScrollViewDelegate {
     /// Will be invoked with the cropped and scaled image and a boolean indicating
     /// whether or not the original image was modified
     var onCompletion: ((UIImage, Bool) -> Void)?
+    var onCancel: (() -> Void)?
     var maskShape: ImageCropOverlayMaskShape = .circle
+    var shouldShowCancelButton: Bool = false
```

Comment:
We try to rely on type inference where we can. No need to specify a `Bool` type when assigning `false` :)